src/main/java/uk/gov/hmcts/reform/ccd/document/am/service/impl/DocumentManagementServiceImpl.java

```diff
@@ -449,18 +451,14 @@ private HttpHeaders getHeaders(ResponseEntity<ByteArrayResource> response) {
 
     public boolean checkUserPermission(ResponseEntity responseEntity, UUID documentId, Permission permissionToCheck) {
         String caseId = extractCaseIdFromMetadata(responseEntity.getBody());
-        if (!ValidationService.validate(caseId)) {
-            LOG.error("Bad Request Exception {}", CASE_ID_INVALID + HttpStatus.BAD_REQUEST);
-            throw new BadRequestException(CASE_ID_INVALID);
```

Comment:
this always threw the exception in ValidationService.validate() so I have removed it